### ISSUE TITLE
very low training accuracies with alexnet or vgg but good with resnet

### ISSUE THREAD

**Issue by amithr1** [apache/incubator-mxnet#5460]
Hi all,

With resnet I get the training accuracies as expected but when I just use some other network, alexnet or vgg, the training accuracies dont change..Not sure why this is,

NFO:root:Epoch[0] Batch [880]	Speed: 231.02 samples/sec	Train-accuracy=0.000391
INFO:root:Epoch[0] Batch [900]	Speed: 234.54 samples/sec	Train-accuracy=0.001563
INFO:root:Epoch[0] Batch [920]	Speed: 232.63 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [940]	Speed: 230.30 samples/sec	Train-accuracy=0.001172
INFO:root:Epoch[0] Batch [960]	Speed: 229.14 samples/sec	Train-accuracy=0.000781
INFO:root:Epoch[0] Batch [980]	Speed: 225.48 samples/sec	Train-accuracy=0.001563
INFO:root:Epoch[0] Batch [1000]	Speed: 233.67 samples/sec	Train-accuracy=0.001953
INFO:root:Epoch[0] Batch [1020]	Speed: 232.90 samples/sec	Train-accuracy=0.001563
INFO:root:Epoch[0] Batch [1040]	Speed: 233.26 samples/sec	Train-accuracy=0.000781
INFO:root:Epoch[0] Batch [1060]	Speed: 230.20 samples/sec	Train-accuracy=0.001563
INFO:root:Epoch[0] Batch [1080]	Speed: 229.51 samples/sec	Train-accuracy=0.000781
INFO:root:Epoch[0] Batch [1100]	Speed: 233.14 samples/sec	Train-accuracy=0.000781
INFO:root:Epoch[0] Batch [1120]	Speed: 227.37 samples/sec	Train-accuracy=0.000391

**Comment by piiswrong** [comment]
@mli

**Comment by anijain2305** [comment]
I am getting the same issue. Were you able to get the right hyper-parameters?

**Comment by mli** [comment]
because vgg and alexnet do not have batchnorm layers, the default weight initializer and learning rate do not fit these two.

**Comment by anijain2305** [comment]
I see. Can you please give the value of  learning rate and weight initializer?
**Default settings**
lr=0.01, lr_factor=0.1, lr_step_epochs='30,60'

Relevant snippet from common/fit.py
    if args.network == 'alexnet':
        # AlexNet will not converge using Xavier
        initializer = mx.init.Normal()


How should these be changed to get better accuracy?

**Comment by Luo-Liang** [comment]
any updates on this? :) I'm getting bad accuracy with alexnet as well after 1 whole epoch. 

2017-07-30 01:36:58,224 Node[0] Epoch[0] Batch [2500]   Speed: 1530.97 samples/sec      Train-accuracy=0.000586

**Comment by xingzhaoo** [comment]
I have the same problem for alexnet. I am using CPU only and running alexnet on asp over 2 workers.
I check the alexnet.py under the symbols directory. The code appears that the original alexnet with 2 columns(in 2 GPUs) are integrated into one. Is that the reason which causes this underfitting  problem? 
```
INFO:root:Epoch[7] Batch [3120]	Speed: 5.91 samples/sec	accuracy=0.078125	top_k_accuracy_5=0.025000
INFO:root:Epoch[8] Batch [2000]	Speed: 6.60 samples/sec	accuracy=0.062500	top_k_accuracy_5=0.015625
INFO:root:Epoch[7] Batch [3140]	Speed: 6.07 samples/sec	accuracy=0.056250	top_k_accuracy_5=0.025000
INFO:root:Epoch[8] Batch [2020]	Speed: 6.55 samples/sec	accuracy=0.078125	top_k_accuracy_5=0.031250
INFO:root:Epoch[7] Batch [3160]	Speed: 6.24 samples/sec	accuracy=0.062500	top_k_accuracy_5=0.006250
```

**Comment by jiecaoyu** [comment]
Using pytorch but has the same problem. Changing the initial lr to 0.01 can probably fix the problem. 0.1 is too high for AlexNet without BatchNorm.

**Comment by piiswrong** [comment]
As @jiecaoyu suggested I think this is due to the learning rate being to large. Try learning_rate=0.01.

Feel free to reopen this issue if problem persists

**Comment by rahul003** [comment]
@anijain2305 @Luo-Liang @xingzhaoo were you able to train alexnet successfully? If so, do you remember what was the issue?
